src/common/util/include/openvino/util/log.hpp

```diff
@@ -85,36 +83,32 @@ class Logger {
 
 void default_logger_handler_func(const std::string& s);
 
-#ifndef PROJECT_ROOT_DIR
```

Comment:
Should we remove: https://github.com/openvinotoolkit/openvino/blob/master/src/CMakeLists.txt#L6 ?

Reply:
Removed. Is not required any more.